jiant/preprocess.py

```diff
@@ -261,6 +263,7 @@ def _build_vocab(args: config.Params, tasks: List[Task], vocab_path: str):
     for task in tasks:  # add custom label namespaces
         # TODO: surface more docs for add_task_label_vocab:
         add_task_label_vocab(vocab, task)
+
```

Comment:
?

Reply:
This was from black. I tried reverting it and black just added it back in.